tests/atoms/AlphaConvertUTest.cxxtest

```diff
@@ -44,17 +44,16 @@ public:
 
 	void tearDown() {}
 
-	// void test_untyped_variable();
-	// void test_variable();
-	// void test_body();
-	// void test_atomspace();
-	// void test_bindlink();
-	// void test_rand_alpha_conversion();
-	// void test_names_alpha_conversion();
-	// void test_vardecl_alpha_conversion();
-	// void test_vardecl_bindlink_alpha_conversion();
-	// void test_values_alpha_conversion();
```

Comment:
if you just change the first letter to not be "t" then cxxtest doesn't think they are tests, and won't run them.  This might be easier than commenting out code.